content/docs/user-guide/managing-external-data.md

```diff
@@ -33,25 +31,32 @@ Currently, the following types (protocols) of external outputs (and
 - HDFS
 - Local files and directories outside the <abbr>workspace</abbr>
 
-> Note that these are a subset of the remote storage types supported by
-> `dvc remote`.
+External outputs require an
+[external cache](/doc/use-cases/shared-development-server#configure-the-external-shared-cache)
+in the same external/remote file.
+
+> Note that [remote storage](/doc/command-reference/remote) is a separate
+> feature, and that external outputs are not pushed or pulled from/to DVC
+> remotes.
 
-> Avoid using the same [DVC remote](/doc/command-reference/remote) (used for
-> `dvc push`, `dvc pull`, etc.) for external outputs, because it may cause file
-> hash overlaps: the hash of an external output could collide with a hash
-> generated locally for another file with different content.
+> ⚠️ Avoid using the same DVC remote used for `dvc push`, `dvc pull`, etc. for
+> external outputs, because it may cause data collisions: the hash of an
+> external output could collide with that of a local file with different
+> content.
 
 ## Examples
 
-Let's take a look at:
+Let's take a look at the following operations on all the supported location
+types:
 
-1. Adding a `dvc remote` to use as cache for data in the external location, and
+1. Adding a `dvc remote` in the same location as the desired outputs, and
    configure it as external <abbr>cache</abbr> with `dvc config`.
```

Comment:
as an external cache?

Reply:
No "an" needed. Maybe "the" but it's OK like this too, I think.